api/routes/wishlistRouter.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+
+const { wishlistController } = require("../controllers");
+
+const router = express.Router();
+
+router.get("/list", wishlistController.listWishlist);
```

Comment:
```suggestion
router.get("", wishlistController.listWishlist);
```
한 번더 `/list`라고 분기시켜주지 않아도 아래 처럼 직관적인 엔드포인트 설계가 될 수 있을 것 같습니다.

```shell
$ http -v GET 127.0.0.1:3000/wishlist
```
```